backend/inspirehep/search/aggregations.py

```diff
@@ -202,3 +202,34 @@ def hep_collection_aggregation(order, title="Collection", agg_type="checkbox"):
             "meta": {"title": title, "order": order, "type": agg_type},
         }
     }
+
+
+def hep_rpp(order, title="Exclude RPP", agg_type="checkbox"):
+    return {
+        "rpp": {
+            "filters": {
+                "filters": {
+                    "Exclude Review Of Particle Physics": {
+                        "bool": {
+                            "must_not": [
+                                {
+                                    "match": {
+                                        "titles.full_title": {
+                                            "query": "Review Of Particle Physics",
```

Comment:
```suggestion
                                            "query": "RPP",
```
your current search is wrong, it matches, e.g. https://labs.inspirehep.net/literature/1714863. We put `RPP` as an extra title for the Review of Particle Physics, so that works.

Reply:
yeah, I knew the query was matching other things as well but I didn't know about the extra ```RPP``` in the title :+1: